tools/licenses/README

```diff
@@ -1,8 +1,17 @@
-To update the sky_engine LICENSE file, run the following in this directory:
+To update the golden license files, run the following in this directory:
 
 ```
 pub get
 gclient sync
-dart lib/main.dart ../../.. > ../../sky/packages/sky_engine/LICENSE
-dart --checked lib/main.dart ../../.. > ../../travis/licenses.golden
+dart lib/main.dart --src ../../.. --out ../../../out/licenses --golden ../../travis/licenses_golden
+```
+
+Then copy any affected files from `../../../out/licenses` to `../../travis/licenses_golden` and add them to your change.
+
+To update the license file used for product releases, run:
```

Comment:
It should be updated any time the golden file changes in a way that involves changes to anything other than the FILE lines. In other words, whenever the actual licenses that apply to each package change.